src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
Please ensure that the `BalanceOperation` class adheres to the SOLID principles. It seems to be well-focused and encapsulated, but always review your classes to ensure they follow these principles for maintainability and extensibility.